scripts/spectrum-vars.js

```diff
@@ -138,6 +138,7 @@ const processTypography = async (
 // TODO: use resolve package to find node_modules
```

Comment:
nit: on a side note, we could tackle this TODO and consider using `resolve` to dynamically locate the `node_modules` directory instead of hardcoding the paths below. Not sure if it is very needed, but I think this could help with symlinked packages.

Reply:
Yeah, I thought about that as well, but opted not to in keeping with my "lightest touch" approach to this PR. Let me know if you feel strongly about making this change now and we can discuss.

Reply:
I actually went ahead and took care of this when incorporating the other change you suggested. 🙌

Reply:
I see, thanks! 😄 And who knows - maybe it actually will make our future selves happy someday!